### ISSUE TITLE
aria-label on new elements

### ISSUE THREAD

**Issue by tomcrane** [UniversalViewer/universalviewer#533]
**UV version:**
```
 universalviewer@3.x
```

**I'm submitting a:**

- [x] bug report
- [ ] feature request => please use the [user stories repo](https://github.com/UniversalViewer/user-stories)
- [ ] support request => Please do not submit support requests here, use [stackoverflow](http://stackoverflow.com/questions/tagged/iiif)

**Current behavior:**
`aria-label` missing for new AV elements

**Expected behavior:**
`aria-label` needs to be added to elements.

**Comment by tomcrane** [comment]
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_button_role

See "Labelling buttons" section; suggests `aria-label` is not required.

Stanford aims for WCAG 2.0
https://www.w3.org/TR/WCAG20/

Stanford's UV work changed anchor tags to buttons.

**Comment by tomcrane** [comment]
From Community Call - Stanford happy to work with UV adopters to agree on a common approach to accessibility, National Library of Wales also. Jennifer Vine from Stanford will be at UVCON so ideal chance on agree what standards are being adhered to and how (as implementation of, say, WCAG 2.0 is not a simple matter of following a recipe book).

**Comment by kzadrozny** [comment]
## aria-label
In creating this common approach, it should be clarified that buttons that have no text, like icon buttons, should have the `aria-label` since there is no text inside the button for the screen reader.

## tabindex
Additionally, since these are buttons, `tabindex` is not needed. The `tabindex` item only needs to be used if a span is given the role button, such as: `<span role="button" tabindex="0"`
https://developer.paciellogroup.com/blog/2014/08/using-the-tabindex-attribute/

### An example would be
```
<div class="nextOptions">
   <button class="btn imageBtn next" aria-label="Next Image">
       <i class="uv-icon-next" aria-hidden="true"></i>
   </button>
   <button class="btn imageBtn last" aria-label="Last Image">
       <i class="uv-icon-last" aria-hidden="true"></i>
   </button>
</div>
```

**Comment by LlGC-szw** [comment]
All issues will be triaged for further investigation or closure by the 28 September 2023. If your issue is still relevant and would like for it be investigated further please comment by 14 September 2023.

**Comment by demiankatz** [comment]
Marked as active until we can confirm that accessibility needs are met appropriately.

**Comment by demiankatz** [comment]
I have no objection to the proposed button format listed here. I'm not sure if there's really any action required on this issue beyond agreeing to that approach. I'd suggest that converting more interactive divs to proper buttons as proposed in #1084 sounds like a good idea; I wonder if it would make sense to close this as superseded by other newer tickets, and then focus on doing the work there.

**Comment by demiankatz** [comment]
Discussed on today's sprint stand-up; no objections to this format, so I'm closing this issue -- no further action needed.
